Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -57,136 +40,94 @@ public static void UnhookEvents()
             lock (ThreadLock)
             {
                 User32.UnhookWinEvent(_eventHandle);
-                foreach (var info in TrackedWindows.Values)
-                {
-                    info.Subclass.FocusChange -= FocusDispatcher;
-                    info.Subclass.Dispose();
-                }
+                Subclasses.Dispose();
                 VBEEvents.Terminate();
+                _vbe.Dispose();
```

Comment:
I don't think this is appropriate. If you [look here](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Main/Extension.cs#L237), you can see we are still using the `_vbe` after we call the `UnhookEvents`. I'm not even sure if it's appropriate to dispose the `_vbe`, since it was handed to us from the addin.

Reply:
I can only agree. Please do not dispose the VBE instance passed in via `HookEvents`. It is managed by the caller of that method, i.e. by the `Extension` class, our entry and exit point.

Reply:
Not sure what I was thinking there. Removed.